README.md

```diff
@@ -23,8 +21,14 @@ $ npm install
 ```
 
 ### Select a Network
+
 The example connects to a local Solana cluster by default.
 
+In order to enable on-chain Move loader/VM logs edit the local network script
+- Edit `node_modules/@solana/web3.js/bin/localnet.sh`
```

Comment:
Why not just encourage `export RUST_LOG=solana=info,solana_move_loader_api=trace,solana_runtime=debug` before running `localnet.sh`?  Editing stuff in node_modules/ is sketchy

Reply:
Good point, a lot easier